roles/matterbridge/templates/matterbridge.toml

```diff
@@ -157,21 +262,75 @@ enable=true
     account="slack.{{ default_slack_team_name }}"
     channel="{{ matterbridge_config.rit.slack.channel }}"
 
+[[gateway]]
+name="gateway_rit_foss_infra"
+enable={{ matterbridge_config.rit_foss_infra.enabled }}
+
+    [[gateway.inout]]
+    account="irc.{{ default_irc_network_name }}"
+    channel="{{ matterbridge_config.rit_foss_infra.irc.channel }}"
+
+    [[gateway.inout]]
+    account="discord.{{ matterbridge_networks.fossrit_discord.network_name }}"
+    channel="{{ matterbridge_config.rit_foss_infra.discord.channel }}"
+
+    [[gateway.inout]]
+    account="telegram.{{ matterbridge_networks.fossrit_telegram.network_name }}"
+    channel="{{ matterbridge_config.rit_foss_infra.telegram.channel }}"
+
+    [[gateway.inout]]
+    account="matrix.{{ matterbridge_networks.fossrit_discord.network_name }}"
+    channel="{{ matterbridge_config.rit_foss_infra.matrix.channel }}"
+
 [[gateway]]
 name="gateway_rit_foss"
-enable=true
+enable={{ matterbridge_config.rit_foss.enabled }}
 
     [[gateway.inout]]
     account="irc.{{ default_irc_network_name }}"
     channel="{{ matterbridge_config.rit_foss.irc.channel }}"
 
+    ; [[gateway.inout]]
+    ; account="slack.{{ default_slack_team_name }}"
+    ; channel="{{ matterbridge_config.rit_foss.slack.channel }}"
```

Comment:
Are you meaning to comment these lines out? Don't you need to use `#` instead of `;` for Matterbridge?

Reply:
is this comment entirely based on github's syntax highlighter showing it in red?

Reply:
That, and all the other comments in this file already use hashes.